prime-router/src/main/kotlin/azure/EmailSenderFunction.kt

```diff
@@ -104,44 +100,23 @@ class EmailSenderFunction {
     ): HttpResponseMessage {
         val logger: Logger = context.logger
         val ret = request.createResponseBuilder(HttpStatus.UNAUTHORIZED)
-        val jwtToken: String = request.headers["authorization"] ?: ""
-        /* Jwt authorization check */
-        if (verifyFromSource(jwtToken, logger) !== null) {
-            /* Body existence check */
-            if (request.body === null) return ret.status(HttpStatus.BAD_REQUEST).build()
-
-            /* Body shape check */
-            val body: TosAgreementForm = parseBody(request.body!!, logger)
-                ?: return ret.status(HttpStatus.BAD_REQUEST).build()
-            if (!body.validate(logger)) return ret.status(HttpStatus.BAD_REQUEST).build()
-
-            logger.info(request.body)
-
-            /* Body exists and has all required properties; sendMail's response decides the outcome of this */
-            val sendgridId: String? = SecretHelper.getSecretService().fetchSecret("SENDGRID_ID")
-            val mail: String = createMail(body)
-            ret.status(sendMail(mail, sendgridId, logger)) /* Status becomes whatever SendGrid returns */
-        } else {
-            logger.info("You are unauthorized to call this endpoint")
-//            logger.info(authBody.toString())
-        }
-        return ret.build()
-    }
 
-    /*INFO:
-    *  This function is returning a DecodedJWT from com.auth0.jwt.interfaces rather than
-    *  a boolean, despite its singular implementation right now, so the DecodedJWT can
-    *  be used in future places that this is called, such as to get claims.
-    */
-    private fun verifyFromSource(jwt: String, logger: Logger): DecodedJWT? {
-        return try {
-            val algorithm: Algorithm = Algorithm.HMAC256(System.getenv("TokenSigningSecret"))
-            val verifier: JWTVerifier = JWT.require(algorithm).withIssuer("reportstream").build()
-            return verifier.verify(jwt)
-        } catch (ex: Throwable) {
-            logger.warning("There was an error while verifying your JWT: ${ex.message}")
-            null
-        }
+        /* Body existence check */
+        if (request.body === null) return ret.status(HttpStatus.BAD_REQUEST).build()
+
+        /* Body shape check */
+        val body: TosAgreementForm = parseBody(request.body!!, logger)
+            ?: return ret.status(HttpStatus.BAD_REQUEST).build()
+
+        /* Body content check */
+        if (!body.validate(logger)) return ret.status(HttpStatus.BAD_REQUEST).build()
+        logger.info(request.body)
+
+        /* Body exists and has all required properties; sendMail's response decides the outcome of this */
+        val sendgridId: String? = SecretHelper.getSecretService().fetchSecret("SENDGRID_ID")
```

Comment:
nit: if the SENDGRID_ID is not set, we could log a message here. Not sure what message sendgrid sends back when this fails, but may make local development easier to debug if that's the problem

Reply:
We're actually doing that down in `sendMail()`! 😎